examples/perf/package.json

```diff
@@ -8,7 +8,9 @@
     "perf": "node perf.js"
   },
   "dependencies": {
-    "shelljs": "^0.8.1",
+  },
+  "devDependencies": {
```

Comment:
This is just example.

`yarn audit --groups dependencies` do not show info about the dependencies.